### ISSUE TITLE
Latest master release?

### ISSUE THREAD

**Issue by vinitkumar** [divio/aldryn-newsblog#529]
When is the latest master going to be released? It contains the new format for the URLS(without dollar `$` sign). Please let us know

**Comment by FinalAngel** [comment]
@vinitkumar are you using master actively? If it's tested and okay I could release it.

**Comment by vinitkumar** [comment]
@FinalAngel Nope, but I can give it a try. I am using the latest stable release on PyPI.

**Comment by FinalAngel** [comment]
If it gets tested and is okay let me know @vinitkumar . You could also use the git master version for a while before we release.

**Comment by vinitkumar** [comment]
@FinalAngel Sure, I am testing it out and will reply on this thread if I feel it's stable enough or will report bugs.

**Comment by vinitkumar** [comment]
@FinalAngel There are some issues with the urls, especially the links on blogs list.
